app/Policies/UserPolicy.php

```diff
@@ -16,7 +16,19 @@ class UserPolicy
 
     public function manage(User $user): bool
     {
-        return $this->requireAdministrativePrivileges($user);
+        return $user->hasAnyRole([
+            // admins
+            Role::ROOT,
+            Role::ADMINISTRATOR,
+
+            // moderation
+            Role::MODERATOR,
+
+            // staff developers
+            Role::CODE_REVIEWER,
+            Role::DEV_COMPLIANCE,
+            Role::QUALITY_ASSURANCE,
```

Comment:
What does "manage user" policy give to these roles? I can't seem to add or remove roles as a Developer with a QA role. It does seem to open up the Moderate tools on the non-admin user page, which doesn't seem desirable for these roles.

Reply:
This is cruft from a change that existed earlier in the branch that I ultimately decided against and removed (a promote/demote button on the manage user infolist). I've reverted this policy change in latest.